modules/migration-manual-rollback.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+// migration/migrating-3-4/troubleshooting.adoc
+// migration/migrating-4-4/troubleshooting.adoc
+// migration/migrating-4-4/troubleshooting.adoc
+[id='migration-manual-rollback_{context}']
+= Manual rollback
+
+You can manually roll back a failed migration. Manual rollback is only required if an application has been quiesced, to prevent data corruption in the PV.
+
+This procedure is not required if the application was not quiesced, because the original application is still running on the source cluster.
+
+.Procedure
+
+. On the target cluster, switch to the migrated project:
++
+----
+$ oc project <project>
+----
+
+. Get the deployed resources:
++
+----
+$ oc get all
+----
+
+. Delete each deployed resource type or scale it to `0`:
```

Comment:
Please present only the best one. If there are different ones that users should do for different reasons, then present them as options in an unordered list with the reasons why you'd want to do one over the other.

Reply:
They're different options. I was trying to save space because it seemed silly to say "Delete resources if you want to delete them" and "Scale resources to 0 if you don't want to delete them." Will do.

Reply:
On second thought, who cares? They can all be deleted. If a failed migration is being rolled back, it doesn't matter whether they're deleted or scaled to 0. :-)